dummyserver/server.py

```diff
@@ -30,6 +35,20 @@
 DEFAULT_CERTS = {
     'certfile': os.path.join(CERTS_PATH, 'server.crt'),
     'keyfile': os.path.join(CERTS_PATH, 'server.key'),
+    'cert_reqs': CERT_OPTIONAL,
+    'ca_certs': os.path.join(CERTS_PATH, 'cacert.pem'),
```

Comment:
DEFAULT_CERTS is passed into tornado to init the SSL context.  We can't actually pass an SSL Context itself because that support wasn't added until 2.7.9, iirc.

The cert_reqs argument sets up the SSL Context to allow client certificate authentication, but not require it - you can connect via https without a certificate, but if you present a certificate, it needs to be validated.  ca_certs is used to contain the root CA for validation of the client certs - anything that can chain back to this root is considered A-OK.

I'll work up something for `CHANGES.rst` shortly.